packages/webcomponents/src/components/checkout/checkout-core.tsx

```diff
@@ -64,6 +64,22 @@ export class CheckoutCore {
     this.getCheckout({
       onSuccess: ({ checkout }) => {
         this.checkout = new Checkout(checkout);
+        const { status } = this.checkout;
+        if (status === 'completed' || status === 'expired') {
```

Comment:
Maybe we can just emit all statuses since the integrator will need to check the status to decide how to handle it anyways. It would remove some complexity on our side. What do you think?

Reply:
That's a good ideia!